pyasn1_modules/rfc3709.py

```diff
@@ -21,6 +21,7 @@
 from pyasn1.type import univ
 
 from pyasn1_modules import rfc5280
+from pyasn1_modules import rfc6170
```

Comment:
I can't help but wonder, if it would make sense, eventually, to order the dependencies in a way that later modules would import earlier modules, not the other way round...

My thinking is that, with these RFCs, people probably were building things on the foundations invented earlier.